tests/Jira/ApiTest.php

```diff
@@ -149,6 +149,40 @@ public function testFindVersionByName()
 		);
 	}
 
+	public function testGetResolutions()
+	{
+		$response = '[{"self":"https://test.atlassian.net/rest/api/2/resolution/1","id":"1","description":"A fix for this issue is checked into the tree and tested.","name":"Fixed"},{"self":"https://test.atlassian.net/rest/api/2/resolution/10000","id":"10000","description":"This issue won\'t be actioned.","name":"Won\'t Do"}]';
+
+		$this->expectClientCall(
+			Api::REQUEST_GET,
+			'/rest/api/2/resolution',
+			array(),
+			$response
+		);
+
+		$actual = $this->api->getResolutions();
+
+		$this->assertEquals(array(
+			'1' => array(
+				'self' => 'https://test.atlassian.net/rest/api/2/resolution/1',
+				'id' => '1',
+				'description' => 'A fix for this issue is checked into the tree and tested.',
+				'name' => 'Fixed',
+			),
+			'10000' => array(
```

Comment:
Actual array content can be replaced with `$response[1]` to more explicitly state, that untouched array from API response is returned.